charts/vela-core/templates/defwithtemplate/apply-terraform-provider.yaml

```diff
@@ -116,7 +116,7 @@ spec:
         #ECProvider: {
         	type:   "ec"
         	apiKey: *"" | string
-        	name:   "ec-provider" | string
+        	name:   *"ec-provider" | string
```

Comment:
we shouldn't mix this kind of fix inside a large PR like this.

Reply:
Fix in this PR: https://github.com/kubevela/kubevela/pull/5485

Reply:
great job